daemon/src/routes_maker.rs

```diff
@@ -208,6 +208,63 @@ pub fn index<'r>(_paths: PathBuf, _auth: Authenticated) -> impl Responder<'r, 's
     Ok::<(ContentType, Cow<[u8]>), Status>((ContentType::HTML, asset.data))
 }
 
+#[derive(Debug, Clone, Deserialize)]
+pub struct WithdrawRequest {
+    pub address: bdk::bitcoin::Address,
+    #[serde(with = "::bdk::bitcoin::util::amount::serde::as_btc")]
+    pub amount: bdk::bitcoin::Amount,
+    pub fee: f32,
+}
+
+#[rocket::post("/withdraw", data = "<withdraw_request>")]
+pub async fn post_withdraw_request(
+    withdraw_request: Json<WithdrawRequest>,
+    wallet: &State<Address<wallet::Actor>>,
+    network: &State<Network>,
+    _auth: Authenticated,
+) -> Result<status::Accepted<String>, HttpApiProblem> {
+    tracing::info!(
+        "Withdrawing {:?} to {} with fee {}",
+        withdraw_request.amount,
+        withdraw_request.address,
+        withdraw_request.fee
+    );
```

Comment:
I think this log should move into the wallet so you can omit it from both endpoints.

Reply:
Such a log already exists 🙈

Reply:
Let's remove it from here then?

Reply:
Already gone :)

Reply:
Huh? I still see a "Withdrawing" log in the diff? What am I missing? :)

Reply:
Nothing, you are completely right, I forgot that one :(